lib/ansible/module_utils/network/eos/config/base.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Red Hat
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+"""
+The base class for all eos resource modules
+"""
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+from ansible.module_utils.connection import Connection
+
+
+class ConfigBase(object):
```

Comment:
This file is not required. The class is defined in the `network/common/cfg/base.py` file